bokeh/models/annotations.py

```diff
@@ -573,7 +573,7 @@ class LabelSet(TextAnnotation):
 
     border_line_color = Override(default=None)
 
-    source = Instance(DataSource, default=lambda: ColumnDataSource(), help="""
```

Comment:
was instance of mismatched JS/python defaults (could have been fixed in separate PR but was done here)